### DISCUSSION TITLE
Beta channel in allowedChannels

### DISCUSSION THREAD

**Discussion by jontelang** [sparkle-project/Sparkle#2662] [Q&A] [answered]
Returning `["beta"]` from `allowedChannels` actually means that the allowed channels are `beta` _and_ the `default` channel right? And returning an empty set means _only_ the `default` channel.

I am sure it is the case, but I don't _think_ it explicitly says so anywhere? 

https://sparkle-project.org/documentation/api-reference/Protocols/SPUUpdaterDelegate.html#/c:objc(pl)SPUUpdaterDelegate(im)allowedChannelsForUpdater:

**Comment by zorgiepoo** [comment, marked as answer]
Yes. I guess that is better explained on the publishing docs https://sparkle-project.org/documentation/publishing/#channels
It may indeed be worthwhile to make the API documentation more explicit that the default channel cannot be excluded.

**Reply by jontelang** [reply]
Thank you. I do believe I read that link some time ago, but as I started to implemented it just now I only looked at the documentation and second guessed myself. 

Agreed on making it clear in the documentation, as well.
